app/main/views/services.py

```diff
@@ -214,7 +214,7 @@ def create_new_draft_service(framework_slug):
     lot = request.form.get('lot', None)
 
     if not lot:
-        question = content_loader.get_builder(framework_slug, 'edit_submission').get_question('lot')
+        question = content_loader.get_question(framework_slug, 'services', 'lot')
```

Comment:
Yes, that's one of the two things that break (the other one being lot names disappear in the service tables). Both are pretty hard to avoid without doing multi-step API deploy and since they don't seem too critical in a short window I'm just going to try and deploy things close to each other.